modules/signing_scriptworker/manifests/settings.pp

```diff
@@ -123,5 +123,35 @@
             datadog_api_key          => secret('scriptworker_datadog_api_key'),
             gpg_keyfile              => 'KEY_dep',
         },
+        'appsv-dep' => {
+            worker_type              => 'appsv-signing-dep-v1',
+            worker_group             => 'appsv-signing-dep-v1',
+            taskcluster_client_id    => 'project/mozilla/application-services/releng/scriptworker/signing/dep',
+            taskcluster_access_token => secret('application_services_signing_dep_scriptworker_taskcluster_access_token'),
+            passwords_template       => 'dep-passwords-appsv.json.erb',
+            scope_prefixes           => ['project:mobile:focus:releng:signing:', 'project:mobile:android-components:releng:signing:', 'project:mobile:fenix:releng:signing:', 'project:mobile:reference-browser:releng:signing:', 'project:mozilla:application-services:releng:signing'],
+            sign_chain_of_trust      => false,
+            verify_chain_of_trust    => true,
+            verify_cot_signature     => false,
+            cot_product              => 'application-services',
+            github_oauth_token       => secret('scriptworker_github_oauth_token_staging'),
+            datadog_api_key          => secret('scriptworker_datadog_api_key'),
+            gpg_keyfile              => 'KEY_dep',
+        },
+        'appsv-prod' => {
+            worker_type              => 'appsv-signing-v1',
+            worker_group             => 'appsv-signing-v1',
+            taskcluster_client_id    => 'project/mozilla/application-services/releng/scriptworker/signing/production',
+            taskcluster_access_token => secret('application_services_signing_scriptworker_taskcluster_access_token'),
+            passwords_template       => 'passwords-appsv.json.erb',
+            scope_prefixes           => ['project:mobile:focus:releng:signing:', 'project:mobile:android-components:releng:signing:', 'project:mobile:fenix:releng:signing:', 'project:mobile:reference-browser:releng:signing:', 'project:mozilla:application-services:releng:signing'],
+            sign_chain_of_trust      => true,
+            verify_chain_of_trust    => true,
+            verify_cot_signature     => true,
+            cot_product              => 'application-services',
+            github_oauth_token       => secret('scriptworker_github_oauth_token_production'),
+            datadog_api_key          => secret('scriptworker_datadog_api_key'),
+            gpg_keyfile              => 'KEY_dep',
```

Comment:
Should this be `KEY_dep`?

Reply:
Good catch, I addressed it in a follow-up.